mabwiser/popularity.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# SPDX-License-Identifer: Apache-2.0
+
+from typing import List, Optional, NoReturn
+import numpy as np
+
+from mabwiser.greedy import _EpsilonGreedy
+from mabwiser.utils import Arm, reset
+
+
+class _Popularity(_EpsilonGreedy):
```

Comment:
Nothing in particular, other than most bandits sits on their own including random

Reply:
The other context-free algorithms are separate because they don't inherit from each other, but the nearest neighbors ones and the linear ones are together.

Reply:
I see, so is your recommendation to move it under _Greedy?

Reply:
I think that would make the most sense

Reply:
I can do that, but my only concern would be this: 

if I am a user and I am curious about the implementation behind the popularity algorithm, which is basically us a month ago when we were trying to find a popularity implementation, it would be nice to come here, browse mabwiser folder and go directly to populairty.py.. 

If I am after the implementation of Neighborhood policies.. I can go to neighborhood.py
If I am after the implementation of LinTS/LinUCB policies.. I can go to linear.py

But the fact that I need to go to "greedy.py" for the implementation of popularity is not apparent anywhere.. Unless I do a some form of search or sth. So the connection between greedy and popularity is not naturally there for me. It is only an implementation connection for only those who knows how things are implemented under the hood. 

That's my reasoning to keep somewhat of a mapping between bandits and the files here. Popularity = popularity.py

Thoughts?

Reply:
Sounds reasonable to me